### PR TITLE
feat(movableCoord): add disable/enableInput

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#459 

## Details
- supports disable/enable input

## Preferred reviewers
@netil

### CODE REVIEW

File: src/movableCoord.js [hunk 1]
```diff
@@ -816,6 +817,32 @@ eg.module("movableCoord", [eg, window, "Hammer"], function(ns, global, HM) {
 		},
 
 		/**
+		 * Enables input devices
+		 * @ko 입력 장치를 사용할 수 있게 한다
+		 * @method eg.MovableCoord#enableInput
+		 * @param {HTMLElement|String|jQuery} element An element from which the eg.MovableCoord module is using<ko>eg.MovableCoord 모듈을 사용하는 엘리먼트</ko>
+		 * @return {eg.MovableCoord} An instance of a module itself <ko>자신의 인스턴스</ko>
+		 */
+		enableInput: function(element) {
```
Comment (netil) [inline src/movableCoord.js]: Why element param is required? IMO, it should use binded element.

File: src/movableCoord.js [hunk 2]
```diff
@@ -816,6 +817,32 @@ eg.module("movableCoord", [eg, window, "Hammer"], function(ns, global, HM) {
...
+		 * @ko 입력 장치를 사용할 수 없게 한다.
+		 * @method eg.MovableCoord#disableInput
+		 * @param {HTMLElement|String|jQuery} element An element from which the eg.MovableCoord module is using<ko>eg.MovableCoord 모듈을 사용하는 엘리먼트</ko>
+		 * @return {eg.MovableCoord} An instance of a module itself <ko>자신의 인스턴스</ko>
+		 */
+		disableInput: function(element) {
+			var hammer = this.getHammer(element);
+			hammer && hammer.get("pan").set({ enable: false });
+			return this;
+		},
```
Comment (netil) [inline src/movableCoord.js]: `.enableInput()` and `.disableInput()` are very similar.
It can be made one method to do these jobs.

Comment (sculove) [inline src/movableCoord.js]: ok!

File: src/movableCoord.js (hunk 1 above)
Comment (sculove) [inline src/movableCoord.js]: movableCoord manages many elements. so it is necessary!!!

I accept your advice on offline.
if element is null or undefined, I will add enable/disable function about all hammer instance.
Thanks

Comment (sculove) [inline src/movableCoord.js]: I applied it

File: src/movableCoord.js (hunk 2 above)
Comment (sculove) [inline src/movableCoord.js]: I applied it

File: src/movableCoord.js [hunk 3]
```diff
@@ -816,6 +817,41 @@ eg.module("movableCoord", [eg, window, "Hammer"], function(ns, global, HM) {
 		},
 
 		/**
+		 * Enables input devices
+		 * @ko 입력 장치를 사용할 수 있게 한다
+		 * @method eg.MovableCoord#enableInput
+		 * @param {HTMLElement|String|jQuery} [element] An element from which the eg.MovableCoord module is using<ko>eg.MovableCoord 모듈을 사용하는 엘리먼트</ko>
```
Comment (netil) [inline src/movableCoord.js]: Declare when element is not given.
ex) Target element or dismiss to apply for all binded elements.

Comment (sculove) [inline src/movableCoord.js]: ok I applied it

**Pull request merged**